### PR TITLE
Add some more checks to databricks build scripts

### PR DESCRIPTION
**Pull request by tgravescs**
Add some checks to aid in debugging and catch if someone changes the init scripts

### CODE REVIEW

**Comment by tgravescs** [comment]
build

**Review by jlowe — APPROVED** [review]
Seems fine to me.  I wonder if the script should auto-detect the plugin and cudf jar names that exist in the destination location and simply adapt to use those to match what the init script expects rather than forcing it to fail.

**Comment by tgravescs** [comment]
Please hold off on merging this for a few a build failed with this and I need to see if my check is wrong in the ok case.

Yeah I want to make more automation to this, this was quick and easy check to add for now.

File: jenkins/databricks/build.sh [hunk 1]
```diff
@@ -106,6 +108,16 @@ sudo ln -s /databricks/jars/ $SPARK_HOME/jars || true
 sudo chmod 777 /databricks/data/logs/
 sudo chmod 777 /databricks/data/logs/*
 echo { \"port\":\"15002\" } > ~/.databricks-connect
+if [ `ls $DB_JAR_LOC/rapids* | wc -l` > 1 ]; then
```
Comment (tgravescs) [inline jenkins/databricks/build.sh:111]: used the wrong >, that should be -gt, testing.

**Comment by tgravescs** [comment]
sorry, I had tested that it failed but didn't wait for build to finish on positive case when there weren't extra files.  Fixed great than and tested it.

**Comment by tgravescs** [comment]
build

**Review by jlowe — APPROVED** [review]

**Review by revans2 — APPROVED** [review]

**Pull request merged**